hieradata_aws/integration.yaml

```diff
@@ -228,7 +228,6 @@ grafana::dashboards::application_dashboards:
   email-alert-service: {}
   feedback: {}
   finder-frontend:
-    show_external_request_time: true
     show_memcached: true
```

Comment:
I added this to [common](https://github.com/alphagov/govuk-puppet/pull/10060/files#diff-833c7fe16144bdf8564e6aa04cfb179bR1024) since I think it was duplicated here - I might be wrong though!

Reply:
I think that `grafana::dashboards::application_dashboards` is redefined for Integration, so I'm not sure the Production value matters.

Reply:
Ah right, thanks for pointing that out. I've added rows here too.